src/main/kotlin/domain/MineLayer.kt

```diff
@@ -0,0 +1,11 @@
+package domain
+
+class MineLayer(private val board: Board, private val selector: PositionSelector) {
+    fun layMines(mineNumber: MineNumber, position: Position = Position()) =
+        selector.selectMinePositions(mineNumber, position).forEach { layMine(it) }
```

Comment:
```suggestion
        selector
            .selectMinePositions(mineNumber, position)
            .forEach { layMine(it) }
```
함수 체이닝 컨벤션을 적용하면 좋을 것 같아요

ref. https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls